README.md

```diff
@@ -34,20 +34,23 @@ These dependencies are subject to change so be sure to verify your version of My
 
 Consult the [LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) page on this [Install process](https://github.com/aces/Loris/wiki/Installing-Loris) for more information.
 
-1. Set up LINUX user lorisadmin and create LORIS base directory:
-
-    ```
-    sudo useradd -U -m -G sudo -s /bin/bash lorisadmin
-    sudo passwd lorisadmin
-    su - lorisadmin
-    ```
-
-    <b>Important: All steps from this point forward must be executed by lorisadmin user</b>
-
-    ```
-    sudo mkdir -m 775 -p /var/www/$projectname
-    sudo chown lorisadmin.lorisadmin /var/www/$projectname
-    ```
+1. Set up LINUX user and group lorisadmin and create LORIS base directory:
+
+```bash
+# Create lorisadmin user and group
+sudo groupadd lorisadmin
```

Comment:
not needed as it will be created by the `useradd`

Reply:
```suggestion
```